backend/src/main/java/com/example/dontwastetomuch/game/GameController.java

```diff
@@ -68,4 +81,12 @@ public List<UserGameDTO> getAllMyGames(Principal principal){
                 }).toList();
     }
 
+    @PutMapping("/myGames/update")
+    public void updateGameStats(@RequestBody UserGameDTO userGameDTO, Principal principal){
+        MyUser user = myUserRepo.findById(principal.getName()).orElseThrow();
+        user.getGameData().stream().filter(game -> userGameDTO.getGameId().equals(game.getGameId())).findAny().orElseThrow().setMoney(userGameDTO.getSpentMoney());
+        user.getGameData().stream().filter(game -> userGameDTO.getGameId().equals(game.getGameId())).findAny().orElseThrow().setPlaytime(userGameDTO.getPlaytime());
```

Comment:
Kann es mehrere Spiele zu einer GameId geben? 
Falls es nur eins geben sollte, würde sich findByGameId anbieten

Reply:
ich glaube ganz so einfach ist das nicht, da der User eine gameData an sich dran hat in denen die Infos zu den Spielen liegen und dies eine Liste von spielen sein kann. Das war auch so meine größte Baustelle heute 😄